yury-ortuno/lesson-1/homework-2-Car/src/Car.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Created by Yury Ortuno on 11/13/2018.
+ */
+public class Car {
+    private String mark;
+    private double gas;
+    private boolean isPowerOn;
+    private double gasConsumeByKilometer;
+    private double maxCapacityGas;
+
+    public Car(String mark) {
+        isPowerOn = false;
+        this.mark = mark;
+    }
+
+    public void setGasConsumeByKilometer(double gasConsumeByKilometer) {
+        this.gasConsumeByKilometer = gasConsumeByKilometer;
+    }
+
+    public double getGasConsumeByKilometer() {
+        return gasConsumeByKilometer;
+    }
+
+    public void setMaxCapacityGas(double maxCapacityGas) {
+        this.maxCapacityGas = maxCapacityGas;
+    }
+
+    public double getMaxCapacityGas() {
+        return maxCapacityGas;
+    }
+
+    public boolean powerOn() {
+        isPowerOn = true;
+        return isPowerOn;
+    }
+
+    public boolean powerOff() {
+        isPowerOn = false;
+        return isPowerOn;
+    }
+
+    public void fillGas(double gas) {
```

Comment:
Gas should be added to actual gas quantity you have

Reply:
Done